utilities/mobile_server_pool.py

```diff
@@ -30,6 +55,12 @@ def get_nodes_from_pool_server(num_of_nodes, nodes_os_type, node_os_version, job
     for row in sdk_client.n1ql_query(query):
         doc_id = row["id"]
         is_node_reserved = reserve_node(doc_id, job_name_requesting_node)
+        vm_alive = check_vm_alive(doc_id)
+        if not vm_alive:
+            query_str = "update `{}` set state=\"VM_NOT_ALLIVE\" where meta().id='{}' " \
+                "and state='available'".format(doc_id)
+            query = N1QLQuery(query_str)
+            sdk_client.n1ql_query(query)
```

Comment:
I assume when vm_alive is not true, then this vm will not be added to the pool_list. do you need to add a "continue" operation after this line 63?

Reply:
oh , thanks for noticiing! Good catch.  I made the change.